delivery/create-security-ticket/action.yml

```diff
@@ -4,12 +4,16 @@ description: Create a Security Jira ticket from a Github issue
 inputs:
   jira-host:
     description: 'Jira host'
-    default: 'jira.jahia.org'
+    default: 'support.jahia.com'
     required: true
   jira-project:
     description: 'Jira project'
-    default: 'SECBACKLOG'
+    default: 'SECURITY'
     required: true
+  jira-issuetype:
+    description: 'Jira issue type'
+    default: 'Vulnerability'

```

Comment:
I know we said we should treat any new Epic as having the potential for introducing vulnerabilities (i.e. follow a similar/same process as a vulnerability). Still, I wonder if we should name the type differently (i.e. a different issue type in Jira).

Reply:
The goal of the SECURITY project is to track security incidents, events or vulnerabilities so we need to either match with that or find another way of dealing with the security review of the Epics